Jenkinsfile

```diff
@@ -13,6 +11,13 @@ pipeline {
     }
   }
 
+  parameters {
+    booleanParam(defaultValue: false, description: 'Skip latest upload', name: 'SKIP_LATEST')
```

Comment:
☝️ Let's not split these as this poses new requirements on how the steps works which significantly increases complexity. I'd revert 37f78b3958b0626981c3f8afd06a3aeb0b3351bc